storm-webapp/src/main/java/org/apache/storm/daemon/logviewer/handler/LogviewerLogSearchHandler.java

```diff
@@ -71,6 +78,10 @@
 
 public class LogviewerLogSearchHandler {
     private static final Logger LOG = LoggerFactory.getLogger(LogviewerLogSearchHandler.class);
+    private static final Meter numDeepSearchNoResult = StormMetricsRegistry.registerMeter("logviewer:num-deep-search-no-result");
```

Comment:
Are these metrics useful?

Reply:
I think it's for analyzing invalid requests, but I'm not sure. @kishorvpatil requested it. You may ask him for his intention.